openedx/core/djangoapps/ace_common/apps.py

```diff
@@ -20,6 +20,7 @@ class AceCommonConfig(AppConfig):
                 SettingsType.AWS: {PluginSettings.RELATIVE_PATH: u'settings.aws'},
                 SettingsType.COMMON: {PluginSettings.RELATIVE_PATH: u'settings.common'},
                 SettingsType.DEVSTACK: {PluginSettings.RELATIVE_PATH: u'settings.common'},
+                SettingsType.TEST: {PluginSettings.RELATIVE_PATH: u'settings.common'},
```

Comment:
I don't believe this line is needed since the `test` settings always import `common` settings.  See https://github.com/edx/edx-platform/blob/master/lms/envs/docs/README.rst

Reply:
Thanks for noting that @nasthagiri!